### PR TITLE
[BD-35] PAR-282 feat: change tooltip component according to new design

### PR DESCRIPTION
**Pull request by amalbas**
@abutterworth 
Ticket on Jira: https://openedx.atlassian.net/browse/PAR-282
Add light version tootltip
light-version-tooltip

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @amalbas! I've created [BLENDED-794](https://openedx.atlassian.net/browse/BLENDED-794) to keep track of it in Jira.


When this pull request is ready, tag your edX technical lead.

File: src/Tooltip/_variables.scss [hunk 1]
```diff
@@ -7,10 +7,15 @@ $tooltip-color:                     $white !default;
 $tooltip-bg:                        $black !default;
 $tooltip-border-radius:             $border-radius !default;
 $tooltip-opacity:                   .9 !default;
```
Comment (abutterworth) [inline src/Tooltip/_variables.scss:9]:
```suggestion
$tooltip-opacity:                   1 !default;
```

Comment (abutterworth) [reply src/Tooltip/_variables.scss:9]: After this change, everything else looks good to me

**Review by abutterworth — CHANGES_REQUESTED** [review]
The tooltip opacity should be 1. Everything else looks good to me.

File: src/Tooltip/_variables.scss (hunk 1 above)
Comment (amalbas) [reply src/Tooltip/_variables.scss]: Done

File: src/Tooltip/README.md [hunk 2]
```diff
@@ -10,14 +10,14 @@ notes: |
...
 ---
 
-<p className="lead">
-  This is a pass through component from React-Bootstrap.<br/>
-  <a href="https://react-bootstrap.github.io/components/overlays#tooltips" target="_blank" rel="noopener noreferrer">
-    See React-Bootstrap for documentation.
-  </a>
-</p>
+import get from 'lodash/get';
+import SingleComponentStatus from '../../components/SingleComponentStatus';
```
Comment (abutterworth) [inline src/Tooltip/README.md:14]:
```suggestion
```

Needs removal in the new docs format.

File: src/Tooltip/README.md [hunk 3]
```diff
@@ -10,14 +10,14 @@ notes: |
...
 
-### Basic Usage
+# Tooltip
+
+<SingleComponentStatus frontmatter={props?.pageContext?.frontmatter} />
+
```
Comment (abutterworth) [inline src/Tooltip/README.md:19]:
```suggestion
```

Needs removal in the new docs format

**Review by abutterworth — CHANGES_REQUESTED** [review]
Looks good, just needs a couple changes to the readme so that the doc site will build.

File: src/Tooltip/README.md [hunk 4]
```diff
@@ -36,3 +29,23 @@ notes: |
...
+##### Light version
+
+```jsx live
+<div className="bg-dark-700 p-4">
+  {['top', 'right', 'bottom', 'left'].map((placement) => (
+    <OverlayTrigger
+      key={placement}
+      placement={placement}
+      overlay={
+        <Tooltip variant="tooltip-light" id={`tooltip-${placement}`}>
```
Comment (abutterworth) [inline src/Tooltip/README.md:42]: I missed this earlier, sorry:

`<Tooltip variant="light" />` seems like a better api than `<Tooltip variant="tooltip-light" />` could we change to reflect that?

File: src/Tooltip/index.jsx [hunk 5]
```diff
@@ -1 +1,34 @@
...
+import Tooltip from 'react-bootstrap/Tooltip';
+
+const WrapperTooltip = React.forwardRef(({
+  children,
+  variant,
+  ...props
+}, ref) => (
+  <Tooltip
+    {...props}
+    className={classNames(variant, props.className)}
```
Comment (abutterworth) [inline src/Tooltip/index.jsx:13]:
```suggestion
    className={classNames({ 'tooltip-light': variant === 'light' }, props.className)}
```

**Review by abutterworth — CHANGES_REQUESTED** [review]

File: src/Tooltip/README.md (hunk 4 above)
Comment (jignaciopm) [reply src/Tooltip/README.md]: @abutterworth fixed!

File: src/Tooltip/index.jsx (hunk 5 above)
Comment (jignaciopm) [reply src/Tooltip/index.jsx]: @abutterworth fixed!

**Review by abutterworth — APPROVED** [review]

**Pull request merged**